buildSrc/src/main/kotlin/Extensions.kt

```diff
@@ -104,6 +108,11 @@ fun DependencyHandler.addAndroidTestDependencies(kotlinVersion: String, includeT
     androidTestImplementation(Library.OKHTTP_MOCK_WEB_SERVER)
 }
 
+inline fun BaseExtension.kotlinOptions(block: KotlinJvmOptions.() -> Unit) {
+    (this as ExtensionAware).extensions.getByName<KotlinJvmOptions>("kotlinOptions").block()
```

Comment:
Not sure if it also applies to extensions, but using `getByName` instead of `configure` forces eager evaluation instead of lazy.

Reply:
Now with your [commit](https://github.com/coil-kt/coil/commit/18a3c12db6d3be86f7081b66bc0188443ce2b491), I think it's ok to write like this.

Reply:
Make `kotlinOptions` lazy now like what defined in gradle source
![image](https://user-images.githubusercontent.com/10363352/112759445-93a22980-9025-11eb-9535-075085b33fd3.png)